lib/erb_lint/linters/extra_newline.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module ERBLint
+  module Linters
+    # Detects multiple blank lines
+    class ExtraNewline < Linter
+      include LinterRegistry
+
+      EXTRA_NEWLINES = /(\n{3,})/m
+
+      def offenses(processed_source)
+        matches = processed_source.file_content.match(EXTRA_NEWLINES)
+        return [] unless matches
+
+        offenses = []
+        matches.captures.each_with_index do |_newlines, index|
+          offenses << Offense.new(
```

Comment:
Not related to this PR but it would be great if a linter would had the same API as a Rubocop cop. We could start by adding a `add_offense` method and let each linter declare a `MSG` constant for the offense message

Reply:
I agree, it would be worthwhile